test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ParallelExecutionTests.cs

```diff
@@ -88,25 +85,32 @@ public async Task MaxDegreeOfParallelism_Queues(int batchSize, int maxExpectedPa
             IHost host = new HostBuilder()
                 .ConfigureDefaultTestHost<ParallelExecutionTests>(b =>
                 {
-                    b.AddAzureStorage();
+                    b.AddAzureStorageQueues(o =>
```

Comment:
These tests can fail because T2 storage extensions uses a static lazy variable for isDynamicSku so the dynamic flag that changes across test isn't fully honored:
https://github.com/Azure/azure-sdk-for-net/blob/2103c3801199eeaa0cd8b762b28658f8223ffb02/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/SkuUtility.cs?plain=1#L13-L17